### PR TITLE
Require SCSS files

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

File: jujugui/static/gui/src/app/assets/css/_mixins.scss [hunk 1]
```diff
@@ -1,3 +1,6 @@
+@import './plugins/cloud-vanilla-theme/scss/global-settings';
```
Comment (hatched) [inline jujugui/static/gui/src/app/assets/css/_mixins.scss]: Are imports deduped?

File: jujugui/static/gui/src/app/components/budget-chart/budget-chart.js [hunk 2]
```diff
@@ -4,6 +4,8 @@
 const PropTypes = require('prop-types');
 const React = require('react');
 
+require('./_budget-chart.scss');
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/budget-chart/budget-chart.js]: I don't think we have any duplicate requires, but are they deduped as well?

File: jujugui/static/gui/src/app/assets/css/_mixins.scss (hunk 1 above)
Comment (huwshimi) [reply jujugui/static/gui/src/app/assets/css/_mixins.scss]: The imports only import variables, so when compiled it only uses the values as required.

File: jujugui/static/gui/src/app/components/budget-chart/budget-chart.js (hunk 2 above)
Comment (huwshimi) [reply jujugui/static/gui/src/app/components/budget-chart/budget-chart.js]: They are deduped in the browserify scss builder! But I think you're right that we don't include anything twice anyway.

**Comment by hatched** [comment]
Can you update the changelog plz.

**Comment by hatched** [comment]
QA Issue:

screen shot 2018-11-14 at 7 00 06 pm

The size of the CSS has also increased quite substantially 
From 356KB -> 61.2KB gz
To 396KB -> 63.3KB gz

**Comment by hatched** [comment]
Thanks @huwshimi

**Pull request merged**